audio/portaudio/Portfile

```diff
@@ -26,35 +25,33 @@ master_sites        ${homepage}archives
 distname            pa_stable_v[strsed ${version} {g/\./_/}]
 extract.suffix      .tgz
 
-checksums           rmd160  0a290cea8a102a45b2df9b72046b3eed0ab35b4f \
-                    sha256  8fe024a5f0681e112c6979808f684c3516061cc51d3acc0b726af98fc96c8d57
+checksums           rmd160  e6e5cd3f3cb7469aa17549c189e445d573567e13 \
+                    sha256  f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513
 
 worksrcdir          ${name}
 
 # By default, PortAudio is built as a universal binary.
 # To build a non-universal library for the host architecture,
 # simply use the --disable-mac-universal option with configure.
 # http://www.portaudio.com/trac/wiki/TutorialDir/Compile/MacintoshCoreAudio
-configure.args			\
-    --enable-option-checking	\
-    --enable-cxx		\
-    --disable-mac-universal	\
-    --without-alsa		\
+configure.args      \
+	--disable-mac-universal \
+	--enable-cxx \
+	--without-alsa		\
```

Comment:
Why are you removing "--enable-option-checking"? What's the rationale?

Reply:
I don't remember anymore but it builds fine if I leave it in now, so I will.
One thing I just noticed is the port won't build if I use trace mode... for some reason it appears like trace mode is hiding libSystem.tbd and that causes clang to fail.